src/axolotl/utils/config/models/input/v0_4_1/__init__.py

```diff
@@ -99,6 +100,13 @@ def validate_dpo_beta(cls, dpo_beta):
             LOG.warning("dpo_beta is deprecated, use rl_beta instead")
         return dpo_beta
 
+    @field_validator("evaluation_strategy")
+    @classmethod
+    def validate_evaluation_strategy(cls, evaluation_strategy):
+        if evaluation_strategy is not None:
+            LOG.warning("evaluation_strategy is deprecated, use eval_strategy instead")
+        return evaluation_strategy
```

Comment:
Do we need to add one validation where we remap `eval_strategy = evaluation_strategy` if both aren't set. If both are set, we should error out.